.github/workflows/main.yml

```diff
@@ -147,6 +147,15 @@ jobs:
             packager: >
               cp .github/deps/other/bin/qt*.dll .github/deps/qt/bin &&
               cpack --verbose --config build/CPackConfig.cmake -C Release
+
+          # Windows Qt 6
+          - os: windows-latest
+            qt: 6.4.2
+            # KisTablet crashes on Qt6. Unlike Qt5, it does allow for switching
```

Comment:
I don't see a difference in how it works in your branch anyway, but it could be that the problem occurs because it fails to uninstall the KisTablet handlers correctly or something. I'd leave it like that for now, maybe a Windows dev wants to look into it or I'll decide to set up a Windows build environment in Qt6 sometime and try to debug it there.